### PR TITLE
py-sentry_sdk, py-spotipy, pspotify_dl: New ports

### PR DESCRIPTION
**Pull request by judaew**
#### Description

- py-sentry_sdk: new port
  Sentry SDK for Python. This is the next line of the Python SDK for Sentry, intended to replace the raven package on PyPI.
- py-spotipy: new port
  A light weight Python library for the Spotify 
- spotify_dl: new port
  Downloads songs from any Spotify playlist, album or track.
- py-mutagen: Add PY310 subport

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 11.6 20G165 x86_64
Xcode 13.1 13A1030d

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: multimedia/spotify_dl/Portfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                spotify_dl
```
Comment (reneeotten) [inline multimedia/spotify_dl/Portfile:6]: can you please use the PyPI name for the port, which appears to be `spotify-dl`? Then you'd use `python.rootname spotify_dl` to change to the underscore, which upstream uses for downloads....

File: multimedia/spotify_dl/Portfile [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+long_description    {*}${description}
+homepage            https://github.com/SathyaBhat/spotify-dl
+
+checksums           rmd160  b4306d38b60ee39aaccf570d3761da7ab928f9f7 \
+                    sha256  158c08bc5e3bcccfea8c46bfa19eef880b5fbb57c31ac375ed332ef80c5410a2 \
+                    size    9714
+
+post-extract {
+    copy ${filespath}/requirements.txt ${worksrcpath}
+}
```
Comment (reneeotten) [inline multimedia/spotify_dl/Portfile:26]: can you please report this to upstream that they should include it in their PyPI files?

File: multimedia/spotify_dl/Portfile [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+patchfiles          patch-use-xdg-base-dir.diff
+
+python.default_version 39
+depends_build       port:py${python.version}-setuptools
+
+depends_lib         port:youtube-dl \
+                    port:py${python.version}-mutagen \
+                    port:py${python.version}-spotipy \
+                    port:py${python.version}-sentry_sdk \
+                    port:py${python.version}-peewee
```
Comment (reneeotten) [inline multimedia/spotify_dl/Portfile:37]: can you please sort these alphabetically?

File: python/py-sentry_sdk/Portfile [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-sentry_sdk
```
Comment (reneeotten) [inline python/py-sentry_sdk/Portfile:6]: please use the PyPI filename here: `py-sentry-sdk` and then there is no need for `python.rootname`

**Review by reneeotten — CHANGES_REQUESTED** [review]
@judaew a few comments for your consideration

File: multimedia/spotify_dl/Portfile (hunk 1 above)
Comment (judaew) [reply multimedia/spotify_dl/Portfile]: Done

File: multimedia/spotify_dl/Portfile (hunk 2 above)
Comment (judaew) [reply multimedia/spotify_dl/Portfile]: Done, https://github.com/SathyaBhat/spotify-dl/issues/225

**Review by reneeotten — APPROVED** [review]

**Pull request merged**